### PR TITLE
#856 - Fix NPE

### PR DESCRIPTION
**Pull request by davsclaus**

### CODE REVIEW

File: camel-idea-plugin/src/main/java/com/github/cameltooling/idea/service/extension/camel/JavaCamelIdeaUtils.java [hunk 1]
```diff
@@ -116,7 +116,8 @@ public class JavaCamelIdeaUtils extends CamelIdeaUtils implements CamelIdeaUtils
 
     @Override
     public boolean isCamelFile(PsiFile file) {
-        if (file != null && JavaFileType.INSTANCE.equals(file.getFileType())) {
+        if (file != null && JavaFileType.INSTANCE.equals(file.getFileType())
+            && file instanceof PsiJavaFile) {
             PsiJavaFile javaFile = (PsiJavaFile) file;
```
Comment (essobedo) [inline camel-idea-plugin/src/main/java/com/github/cameltooling/idea/service/extension/camel/JavaCamelIdeaUtils.java:121]:
```suggestion
            && file instanceof PsiJavaFile javaFile) {
```

**Comment by essobedo** [comment]
The failures are due to https://github.com/camel-tooling/camel-idea-plugin/pull/857

**Comment by davsclaus** [comment]
Can we avoid that kubernetes JAR ; what do we really use it for

**Comment by essobedo** [comment]
It is needed for this https://github.com/camel-tooling/camel-idea-plugin/pull/709

**Comment by essobedo** [comment]
@davsclaus I confirm that I get this error very regularly so it is quite annoying. I'm wondering if we could have a new release with this bug fix, WDYT?

**Comment by davsclaus** [comment]
yeah sure

**Pull request merged**